src/less-style/question-props.less

```diff
@@ -157,3 +157,17 @@
   padding: 0 2em;
   margin-bottom: 2em;
 }
+
+.fake-textarea {
+    -moz-appearance: textfield-multiline;
+    -webkit-appearance: textarea;
+    border: 1px solid #cccccc;
+    border-radius: 4px;
+    font: medium -moz-fixed;
+    font: -webkit-small-control;
+    height: 28px;
```

Comment:
I think we should be using font-based dimensions rather than pixels. So this should be something like `1em`.